source/index.rst

```diff
@@ -27,4 +27,5 @@ Git Extensions |release| Manual
   windows_explorer
   other_tools
   command_line
+  manual_settings
```

Comment:
Why not in Settings, parallel to GE/Git/Plugins?

Reply:
> Why not in Settings, parallel to GE/Git/Plugins?

Because this section describes the settings pages GUI.
I did not want to "hide" it there. But I could add another link at the bottom of `settings.rst`.

Reply:
Ok, buy I find it natural to look for this as a hidden page